test/unit/math/mathOps/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2017-2019 Benjamin Worpitz, Jakob Krude
+#
+# This file is part of Alpaka.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+SET(_TARGET_NAME "mathOps")
+
+
+append_recursive_files_add_to_src_group("src/" "src/" "cpp" _FILES_SOURCE)
+
+
+message(STATUS "Disabling fast math options for math operation tests only.")
+if(ALPAKA_ACC_GPU_CUDA_ENABLE)
+    list(REMOVE_ITEM
+            CUDA_NVCC_FLAGS "--ftz=true" "--prec-div=false" "--prec-sqrt=false" "--fmad=true" "--use_fast_math" "-use_fast_math")
+
+endif()
+if(ALPAKA_ACC_GPU_HIP_ENABLE)
+    list(REMOVE_ITEM
+            HIP_NVCC_FLAGS "--ftz=true" "--prec-div=false" "--prec-sqrt=false" "--fmad=true" "--use_fast_math" "-use_fast_math")
+endif()
+
+
+ALPAKA_ADD_EXECUTABLE(
+    ${_TARGET_NAME}
+    ${_FILES_SOURCE})
+TARGET_INCLUDE_DIRECTORIES(
+    ${_TARGET_NAME}
+    PRIVATE ${Boost_INCLUDE_DIRS})
+TARGET_LINK_LIBRARIES(
+    ${_TARGET_NAME}
+    PRIVATE common)
+SET_TARGET_PROPERTIES(
+    ${_TARGET_NAME}
+    PROPERTIES
+    COMPILE_OPTIONS
+    $<$<CXX_COMPILER_ID:clang>:"-ffp-contract=off"> # ffp-contract: https://llvm.org/docs/CompileCudaWithLLVM.html#id5
+    )
+
+SET_TARGET_PROPERTIES(${_TARGET_NAME} PROPERTIES FOLDER "test/unit")
+
+ADD_TEST(NAME ${_TARGET_NAME} COMMAND ${_TARGET_NAME} ${_ALPAKA_TEST_OPTIONS})
```

Comment:
Could you please make the cmake commands (but not the variables) in this file lowercase to match the changed style in all other files?